navigator_api/migrations/alembic.ini

```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
+
+[alembic]
```

Comment:
Is alembic something that is important for me to get my head round for engine in the next 5 days? 

Given that, engine's entire data structure at this time is actually managed and maintained in Google sheets, and then loaded in a single step into the db, I'm hoping this is something we can push back for now?  i.e. for now we just wipe the db clean and reload everything when we need to.

Reply:
That defaul config for alembic used by flask-migration. That's for nav_api db will needs to be persistent and support migrations if we wish to change the db schema in the future.
I think it's alright to go with this vanillia config for this year.